content/en/blog/2024/sig-cloud-provider-spotlight.md

```diff
@@ -0,0 +1,143 @@
+venues.
+
+## Important subprojects
+
+**Arujjwal:** There are a lot of subprojects within this SIG, can you highlight some of the most
+important ones and what job they do?
+
+**Michael:** I think the two most important subprojects today are the [cloud provider
+framework](https://github.com/kubernetes/community/blob/master/sig-cloud-provider/README.md#kubernetes-cloud-provider)
+and the [extraction/migration
+project](https://github.com/kubernetes/community/blob/master/sig-cloud-provider/README.md#cloud-provider-extraction-migration). The
+cloud provider framework is a common library to help infrastructure integrators build a cloud
+controller for their infrastructure. This project is most frequently the starting point for new
+people coming to the SIG. The extraction and migration project is the other big subproject and a
+large part of why the framework exists, a little history might help explain further: for a long
+time, Kubernetes has needed to have some integration with the underlying infrastructure, not
+necessarily to add features but to be aware of cloud events like instance termination. The cloud
+provider integrations were built into the Kubernetes code tree, and thus the term "in-tree" was
+created (check out this [article on the topic](https://kaslin.rocks/out-of-tree/) for more
+info). The activity of maintaining provider-specific code in the main Kubernetes source tree was
+considered undesirable by the community. The community’s decision inspired the creation of the
+extraction and migration project to remove the "in-tree" cloud controllers in favour of
+"out-of-tree" components.
+
+
+**Arujjwal:** What makes it a good place to start? Does it have consistent good beginner work? What
+kind?
+
+**Michael:** I feel that the cloud provider framework is a good place to start as it encodes the
+community’s preferred practices for cloud controller managers, and as such will give a newcomer a
+strong understanding of how and what the managers do. Unfortunately, there is not a consistent
+stream of beginner work on this component, this is due in part to the mature nature of the framework
+and that of the individual providers as well. For folks who are interested in getting more involved,
+having some [Go language](https://go.dev/) knowledge is good, and also having an understanding of
+how at least one cloud API (eg. AWS, Azure, GCP) works is also beneficial. In my personal opinion,
+being a newcomer to SIG cloud provider can be challenging as most of the code around this project
+deals directly with specific cloud provider interactions. My best advice to people wanting to do
+more work on cloud providers is to grow your familiarity with one or two cloud APIs and then look
+for open issues on the controller managers for those clouds, and always communicate with the other
+contributors as much as possible.
+
+## Accomplishments
+
+**Arujjwal:** Can you share about an accomplishment(s) of the SIG that you are proud of?
```

Comment:
```suggestion
**Arujjwal:** Can you share an accomplishment of the SIG that you are proud of?
```

Reply:
Why this change?

Reply:
This may have been a misunderstanding of how deep of an edit folks would like on these types of blogs. I was making it cleaner to read.

Reply:
I'll ping the author here.

Reply:
Author prefers to keep original version.